step_definitions/topic-steps.js

```diff
@@ -0,0 +1,15 @@
+var {When} = require('cucumber');
+var {Then} = require('cucumber');
+const addEditTopicPage = require("../page-objects/pages/coursePages/CRSSAddEditTopicPage");
```

Comment:
var {When} = require('cucumber');
var {Then} = require('cucumber');
remove it, it is not needed, globally access